Source/Scene/GltfLoaderUtil.js

```diff
@@ -201,7 +201,7 @@ var defaultScale = new Cartesian2(1.0, 1.0);
  * @param {String} [options.channels] The texture channels to read from.
  * @param {Texture} [options.texture] The texture object.
  *
- * @returns {ModelComponents.Texture} The texture.
+ * @returns {ModelComponents.TextureReader} The texture.
```

Comment:
I think the documentation is getting a little mixed here because it's saying there's a `texture` and a `texture object`. I think every time we refer to a `TextureReader`, it should be documented as maybe `A object containing information to read the texture` or even simpler, `the texture reader`.